stimela/cargo/base/equolver/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM stimela/base:1.2.5
```

Comment:
Any particular reason for such an old base, @gigjozsa? Current is 1.6.0.

Reply:
Nope. I will update. And just found out why keeping the tags synchronized with the version is not so simple.